sections/futures/ProfitCalculator/ProfitCalculator.tsx

```diff
@@ -131,10 +131,15 @@ const ProfitCalculator: FC<ProfitCalculatorProps> = ({ marketAsset, setOpenProfi
 
 	return (
 		<>
-			<BaseModal
+			<StyledBaseModal
 				onDismiss={() => setOpenProfitCalcModal(false)}
 				isOpen
 				title={t('futures.modals.profit-calculator.title')}
+				resizableAnDraggableProps={{
```

Comment:
This seems to have a typo. I think it should be `resizableAndDraggableProps`. Moreover, it should be fine to just name it `dndProps`.

Reply:
I will name it as `rndProps`  since the component's name is `ResizableAndDraggable`